apps/artist/routes.coffee

```diff
@@ -18,6 +18,7 @@ sd = require('sharify').data
 
 @index = (req, res, next) ->
   tab = if req.params.tab? then req.params.tab else ''
+  showArtistCTA = req.query.show_artist_cta_code is SHOW_ARTIST_CTA_CODE
```

Comment:
It starts as null, but further down in that file it gets set from the environment.